mewarpx/mewarpx/sim_control.py

```diff
@@ -93,3 +98,26 @@ def check_criteria(self):
             logger.info(terminate_statement)
 
         return all_good
+
+    def write_results(self):
+        """Create results.txt file, and write to it if write_func is set.
+        The file signifies that the simulation ran to completion."""
+        results_string = ""
+        if callable(self._write_func):
+            results_string = self._write_func()
```

Comment:
I guess we should also add the following in case a write function is not specified.
```suggestion
        if callable(self._write_func):
            results_string = self._write_func()
        else:
            results_string = ""
```

Reply:
`results_string` is set to an empty string already by default on the line before the `if callable` check

Reply:
Oh sorry I missed that.